index.js

```diff
@@ -1,13 +1,27 @@
-var mapSize = 10;
-var i, j;
+var gridSize = 11;
+var gridCenter = Math.round(gridSize/2);
 
-for(i = 0; i < mapSize; i++){
+var fragment = document.createDocumentFragment();
+for(var i = 1; i <= gridSize; i++){
     var divRow = document.createElement("div");
-    divRow.className = "row row-" + i;
-    document.body.appendChild(divRow);
-    for (j = 0; j < mapSize; j++){
+    divRow.classList.add("row", "row-" + i);
+    for(var j = 1; j <= gridSize; j++){
         var divCell = document.createElement("div");
-        divCell.className = "cell cell-" + j;
-        document.body.appendChild(divCell);
+        divCell.classList.add("cell", "cell-" + j);
+        divRow.appendChild(divCell);
     }
+    fragment.appendChild(divRow);
 }
+gridWrapper.appendChild(fragment);
+
+var rows = Array.from(document.getElementsByClassName("row"));
```

Comment:
You can simplify this logic. 
What you are doing here is following:
1) find middle row
2) find middle cell in it
3) set class active to middle cell and two cells behind it. 
You can achieve firs step by using `document.querySelector`. Please, read about it on MDN. 
You can provide `'.row-' + gridCenter` as an argument (which should be a valid CSS selector to DOM element).
Then, inside this element, you can also query for children. 
You could iterate from `gridCenter - 2` to `gridCenter` and add class to found cell.
```javascript
for (var i = gridCenter - 2; i <= gridCenter; i++) {
  row.querySelector('.cell-' + i)
    .classList.add('activeCell');
}
```